torch_glow/src/CMakeLists.txt

```diff
@@ -56,6 +57,7 @@ target_include_directories(PyTorchFileLoader PUBLIC
                             ${PYTORCH_DIR}/include)
 
 target_include_directories(_torch_glow PUBLIC
-                            ${PYTORCH_DIR}/include)
+                            ${PYTORCH_DIR}/include
+                            ${TORCH_GLOW}/src/training)
```

Comment:
is this also needed in addition to the target_link_libraries(...TorchGlowTraining...) ?

Reply:
yes, because glow doesn't follow fbcode standard for including paths (always from fbcode, even for local files), the additional include directories must be specified for the projects with more than one directory.